dcc-release-resources/src/main/resources/org/icgc/dcc/release/resources/mappings/donor-centric.mapping.json

```diff
@@ -28,9 +28,15 @@
     	          "consequence":{
     	            "type":"nested"
     	          },
-                 "observation":{
-                  "type":"nested"
-                }
+                  "observation":{
+                    "type":"nested"
+                  },
+                  "clinical_significance": {
+                    "type": "object"
+                  },
+                  "clinical_evidence": {
+                    "type": "nested"
```

Comment:
nested -> array?

Reply:
Nested is specifically for an array of objects so that the keys don't get mashed together across the different objects ...

https://www.elastic.co/guide/en/elasticsearch/reference/current/nested.html